DEVELOPER_README.md

```diff
@@ -166,30 +166,32 @@ Setting different developer profiles while building is done by setting the envir
 
 For example: `KANIDM_BUILD_PROFILE=release_suse_generic cargo build --release --bin kanidmd`
 
-### Building the development container
+### Build a kanidm container
 
-Build a development container with the current branch using:
+Build a container with the current branch using:
 
     make <TARGET>
 
 Check `make help` for a list of valid targets.
 
 The following environment variables control the build:
 
-|ENV variable|Definition|
-|-|-|
-|`IMAGE_BASE`|Base location of the container image. Defaults to `kanidm`.|
-|`IMAGE_VERSION`|Determines the container's tag.|
-|`CONTAINER_TOOL_ARGS`|Specify extra options for the container build tool.|
-|`IMAGE_ARCH`|Passed to `--platforms` when the container is built. Defaults to `linux/amd64,linux/arm64`.|
-|`CONTAINER_BUILD_ARGS`|Override default ARG settings during the container build.|
-|`CONTAINER_TOOL`|Use an alternative container build tool. Defaults to `docker`.|
-|`BOOK_VERSION`|Sets version used when building the documentation book. Defaults to `master`.|
+|ENV variable|Definition|Default|
+|-|-|-|
+|`IMAGE_BASE`|Base location of the container image.|`kanidm`|
+|`IMAGE_VERSION`|Determines the container's tag.|None|
+|`CONTAINER_TOOL_ARGS`|Specify extra options for the container build tool.|None|
+|`IMAGE_ARCH`|Passed to `--platforms` when the container is built.|`linux/amd64,linux/arm64`|
+|`CONTAINER_BUILD_ARGS`|Override default ARG settings during the container build.|None|
+|`CONTAINER_TOOL`|Use an alternative container build tool.|`docker`|
+|`BOOK_VERSION`|Sets version used when building the documentation book.|`master`|
 
-Build a development container using `podman`:
+#### Container build examples
+
+Build a `kanidm` container using `podman`:
 
     CONTAINER_TOOL=podman make build/kanidmd
 
-Build a development container and specify a redis build cache:
+Build a `kanidm` container and use a redis build cache:
 
     CONTAINER_BUILD_ARGS="--build-arg "SCCACHE_REDIS=redis://redis.dev.blackhats.net.au:6379"" make build/kanidmd
```

Comment:
Now that I'm re-reading this, the quoting on that last line looks wrong - should it be:

```shell
CONTAINER_BUILD_ARGS='--build-arg "SCCACHE_REDIS=redis://redis.dev.blackhats.net.au:6379"'
```

Reply:
@yaleman good catch, I'll leave this one to you to approve, I have some wasm to go wrestle :) 

Thanks again @kellinm, really appreciate it!

Reply:
It looks weird, but it is correct shell syntax and worked when tested.

Both variants pass muster with `ShellCheck` - though I prefer the double-double quote method because newcomers may try the single quote syntax, but also try to expand a local ENV variable in their invocation, and trip over shell expansion.

Your call as the maintainers, but it does work this way and is valid syntax.

Reply:
I think we should make it the single wrapping on the outer, just to remove abiguity :)

Reply:
Changed. :)